app/src/common/shared/com/igalia/wolvic/browser/engine/Session.java

```diff
@@ -1022,10 +1024,15 @@ private boolean trySetUaMode(int mode) {
         return true;
     }
 
-    public void setUaMode(int mode) {
+    public void setUaMode(int mode, boolean reload) {
```

Comment:
can we use an enum for the second paramenter. Something like Reload::Yes, Reload:No instead?

Reply:
I think in Java it is fine like this? The IDE already shows the name of the parameter when you call this method. I can add the enum if you prefer.